src/js/components/ColorPicker.js

```diff
@@ -0,0 +1,31 @@
+import Control from './Control';
+import ColorPickerReact from '../components-react/ColorPicker';
+import React from 'react';
+import Message from '../constant/Message';
+
+const validEventNames = ['changeComplete', 'accept', 'cancel'];
+export default class ColorPicker extends Control {
+    _reactComponentClass = ColorPickerReact;
+
+    _getReactElement() {
+      const Component = this._reactComponentClass;
+      // eslint-disable-next-line react/jsx-filename-extension
+      const reactElement = <Component {...this.props} />;
```

Comment:
Please, add ref to assign react object to this. _reactObject
The way to have access to react object has been changed in Control class, so when overriding _getReactElement you need to also assign ref, otherwise "on" function which uses "this._reactObject" might not work.

Ref.: https://github.com/kintone/kintone-ui-component/blob/v0.2.0/src/js/components/Control.js#L68

Reply:
I have added ref in _getReactElement and confirmed the "on" function is work well.